engine/antigravity_engine/hub/ask_pipeline.py

```
@@ -526,22 +624,27 @@ async def _ask_with_agent_md(workspace: Path, question: str) -> str | None:
+            answer = await _run_with_optional_stream(
+                agent=answer_agent,
+                prompt=answer_prompt,
+                max_turns=1,
+                timeout=ask_timeout if ask_timeout > 0 else None,
+                stream_enabled=settings.STREAM_ENABLED,
+                progress_label="Generating answer from agent.md...",
+            )
             print("[4/4] Answer synthesized.", file=sys.stderr)
-            return str(result.final_output).strip()
+            return answer
         except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)